packages/frontend-2/pages/dashboard.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="flex flex-col gap-y-12">
+    <section>
+      <h2 class="text-heading-sm text-foreground-2">Quick start</h2>
+      <div class="flex flex-col md:grid md:grid-cols-3 gap-3 pt-5">
+        <QuickStartCard
+          v-for="(quickStartItem, index) in quickStartItems"
+          :key="`quick-start-${index}`"
+          :title="quickStartItem.title"
+          :description="quickStartItem.description"
+          :buttons="quickStartItem.buttons"
+        />
+      </div>
+    </section>
+    <section>
+      <h2 class="text-heading-sm text-foreground-2">Recently updated projects</h2>
+      <div class="flex flex-col md:grid md:grid-cols-3 gap-3 pt-5">
+        <DashboardProjectCard
+          v-for="project in projects"
+          :key="project.id"
+          :project="project"
+        />
+      </div>
+    </section>
+    <section>
+      <h2 class="text-heading-sm text-foreground-2">News &amp; tutorials</h2>
+      <div
+        class="flex flex-col gap-y-3 md:grid md:grid-cols-2 lg:grid-cols-4 gap-3 pt-5"
+      >
+        <DashboardTutorialCard
+          v-for="tutorial in tutorials"
+          :key="tutorial.id"
+          :tutorial="tutorial"
+        />
+      </div>
+    </section>
+  </div>
+</template>
+<script setup lang="ts">
+import { dashboardProjectsQuery } from '~~/lib/dashboard/graphql/queries'
+import type { TutorialCard, QuickStartItem } from '~~/lib/dashboard/helpers/types'
+import { useQuery } from '@vue/apollo-composable'
+import { useMixpanel } from '~~/lib/core/composables/mp'
+import GhostContentAPI from '@tryghost/content-api'
+import { docsPageUrl, forumPageUrl } from '~~/lib/common/helpers/route'
+import type { ManagerExtension } from '~~/lib/common/utils/downloadManager'
+import { downloadManager } from '~~/lib/common/utils/downloadManager'
+
+useHead({ title: 'Dashboard' })
```

Comment:
We should probably hide this behind a feature flag, if we don't want this to be accessible. Even the workspaces flag would work. You can return a 404 in an inline middleware.

Reply:
@fabis94 I can add this, but I dont really think we need to specifically hide this? It's not depended on anything workspace related, and you can only get there when know the URL (which is also fine, as it's a functional page)

Reply:
Ok, if its fine for unsuspecting visitors and search indexers to pick up on it, then its fine to leave it this way